oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandlerTest.java

```diff
@@ -2386,6 +2427,15 @@ private void withExistingSessionAuthenticatedValue(Boolean isAuthenticated) {
                                 new Session("a-new-session-id").setAuthenticated(isAuthenticated)));
     }
 
+    private void withExistingSessionCurrentCredentialStrengthValue(
```

Comment:
I wonder if rather than defining this it's neater to use `withExistingSession(new Session(NEW_SESSION_ID).setCurrentCredentialStrength(credentialStrength))` which is more flexible?

Reply:
I think i would say its a bit neater to wrap it in a function with detailed naming but since its only used once happy to make this change.